test/fixtures/embedded-in-markdown/output.md

```diff
@@ -59,9 +59,7 @@ For best results, you should only have one `<style>` tag per-Astro component. Th
       ...;
     </style>
   </head>
-  <body>
-    ...
-  </body>
+  <body> ... </body>
```

Comment:
We need to implement the logic for when the content of an element needs to break.